src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderTable/PaymentBuilderTable.tsx

```diff
@@ -147,13 +152,14 @@ const useGetPaymentBuilderColumns = ({
           ]
         : []),
     ],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are we disabling exhaustive deps here? I think we should not do it when possible, or just write a comment why. Most of the times, we want to keep this rule on.